.github/workflows/release-winget.yml

```diff
@@ -17,7 +24,17 @@ jobs:
           $github.release.tag_name -match '\d.*'
           $version = $Matches[0] -replace ".vfs",""
 
-          # Download and run wingetcreate
+          # Download wingetcreate and create manifests
           Invoke-WebRequest https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
-          .\wingetcreate.exe update Microsoft.Git -u $asset.browser_download_url -v $version -o manifests -t "${{ secrets.WINGET_TOKEN }}" -s
+          .\wingetcreate.exe update Microsoft.Git -u $asset.browser_download_url -v $version -o manifests
+
+          # Manually substitute the name of the default branch in the License
+          # and Copyright URLs since the tooling cannot do that for us.
+          $shortenedVersion = $version -replace ".{4}$"
+          $manifestPath = dir -Path ./manifests -Filter Microsoft.Git.locale.en-US.yaml -Recurse | %{$_.FullName}
```

Comment:
This is some fun script magic. It looks sensible, but we will only be sure once we run it for real.

Reply:
Welp. I did run it for real yesterday lol. I tried to write to the [test repo](https://github.com/microsoft/winget-pkgs-test),  but the config failed and submitted a PR to the real repo instead, which I closed immediately. You can check it out here though.

https://github.com/microsoft/winget-pkgs/pull/95353/files

Reply:
Ah, I see that it wrote `https://github.com/microsoft/git/blob/vfs-2.39.10/COPYING` (note `.10` instead of `.1`).

Reply:
or, perhaps you had created a `v2.39.10.vfs.0.5` tag, based on the rest of the context. Carry on.

Reply:
Yes, that was one of the experimental tags I created for testing!